homeassistant/components/sms/config_flow.py

```diff
@@ -27,7 +27,7 @@ async def get_imei_from_config(hass: core.HomeAssistant, data):
         raise CannotConnect
     try:
         imei = await gateway.get_imei_async()
-    except gammu.GSMError as err:  # pylint: disable=no-member
```

Comment:
Did this slip in?

Reply:
Yes, but still passed, so should not have been ignored in the first place.

![image](https://user-images.githubusercontent.com/195327/107820306-ddd39200-6d7a-11eb-9105-c1040660b827.png)